library/src/main/java/com/okta/oidc/clients/web/WebAuthClient.java

```diff
@@ -156,4 +157,49 @@ void registerCallback(ResultCallback<AuthorizationStatus, AuthorizationException
      * @param data        the redirected data from chrome custom tab.
      */
     void handleActivityResult(int requestCode, int resultCode, Intent data);
+
+    /**
+     * Convenience method to completely sign out of application.
+     * Performs the following operations in order:
+     * 1. Revokes the access_token. If this fails step 3 will not be attempted.
+     * 2. Revokes the refresh_token. If this fails step 3 will not be attempted.
+     * 3. Removes all persistence data. Only if steps 1 & 2 succeeds.
+     * 4. Clears browser session only if this is a {@link com.okta.oidc.clients.web.WebAuthClient}
+     * instance. Will always be attempted regardless of the other operations.
```

Comment:
Is it the same as in js implementation? Or it is android specific implementation?

Reply:
It is the same as the js implementation. 

https://github.com/okta/oss-technical-designs/blob/master/technical_designs/rp-initiated-logout.md#mobile